config/locales/application_states.yml

```diff
@@ -297,7 +297,7 @@ en:
       description: The provider says the candidate hasn’t met the conditions set out in the offer.
       emails:
         - candidate_mailer-conditions_not_met
-        - provider_mailer-application_withrawn
```

Comment:
I think was misplaced and it belonged in the transition below?

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/995af4fd81ae7411d85640d63933445f9b5577d2/config/locales/application_states.yml#L301-L304

Reply:
good spot, will put it there